lesson_08/app2/app2.go

```diff
@@ -0,0 +1,64 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+	"time"
+)
+
+var (
+//name    = ""
+//address = ""
+)
+
+func handler(w http.ResponseWriter, r *http.Request) {
+	// send Not found in such case
+	if r.URL.Path != "/" {
+		http.NotFound(w, r)
+		return
+	}
+	switch r.Method {
+	case "POST":
+		var result map[string]interface{}
+		json.NewDecoder(r.Body).Decode(&result)
+
+		log.Println(result["token"])
+
+		currentDate := time.Now()
+
+		message := map[string]interface{}{
```

Comment:
you need unmarshal structrure here from body. And store it in map in global scope